akka-docs/src/main/paradox/multi-jvm-testing.md

```diff
@@ -6,20 +6,26 @@ Useful for integration testing where multiple systems communicate with each othe
 ## Setup
 
 The multi-JVM testing is an sbt plugin that you can find at [https://github.com/sbt/sbt-multi-jvm](https://github.com/sbt/sbt-multi-jvm).
+To configure it for your project you should do the following steps:
 
-You can add it as a plugin by adding the following to your project/plugins.sbt:
+1. Add it as a plugin by adding the following to your project/plugins.sbt:
 
-@@snip [plugins.sbt]($akka$/project/plugins.sbt) { #sbt-multi-jvm }
+    @@snip [plugins.sbt]($akka$/project/plugins.sbt) { #sbt-multi-jvm }
 
-You can then add multi-JVM testing to `build.sbt` or `project/Build.scala` by including the `MultiJvm`
-settings and config. Please note that MultiJvm test sources are located in `src/multi-jvm/...`,
-and not in `src/test/...`.
-
-You can specify JVM options for the forked JVMs:
+2. Add multi-JVM testing to `build.sbt` or `project/Build.scala` by including the `MultiJvm` 
+settings and config.
 
-```
-jvmOptions in MultiJvm := Seq("-Xmx256M")
-```
+    ```none
+    import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
+    import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
+    
+    lazy val root = (project in file("."))
+      .configs(MultiJvm)
+      .settings(multiJvmSettings)
```

Comment:
This is a bit dated. sbt-multi-jvm is an AutoPlugin right now. So instead of `.settings(multiJvmSettings)` the plugin should be enabled `enablePlugins(MultiJvmPlugin)` and then all of the needed settings will be added by the plugin automatically.

`.configs(MultiJvm)` is still needed anyway.

Reply:
Good catch! I'll fix it!